dcos/recordio.py

```diff
@@ -0,0 +1,155 @@
+"""
```

Comment:
Yes. I wasn't sure how to exclude them from this PR though since they aren't merged into master yet. This PR is rebased directly on top of that PR though.